jupyterlab-mlops/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM jupyter/pyspark-notebook:lab-3.1.9
+
+USER root
+
+# allow passwordless sudo and add group required by kubeflow
+RUN echo "jovyan ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers.d/notebook && \
+    groupadd -g 1337 nbusers
+
+# copy mlflow logo and start script
+COPY mlflow/start-mlflow.sh /usr/local/bin
+COPY mlflow/logo.svg /usr/local/share/mlflow-logo.svg
+
+USER jovyan
+
+# install git extension
+RUN pip install jupyterlab-git
+
+# add kfp and install mlflow inside jupyterlab
+RUN mamba install --quiet --yes kfp mlflow jupyter-server-proxy -c conda-forge && \
```

Comment:
No fixed version for `mlflow`? Maybe we should put kedro and mlflow versions as docker `ARG`?

Reply:
Fantastic idea, thanks!